resource.address/spi/src/main/java/org/kaazing/gateway/resource/address/ResourceAddressFactorySpi.java

```diff
@@ -42,7 +45,11 @@
 
 public abstract class ResourceAddressFactorySpi<T extends ResourceAddress> {
 
+    private static final String PREFER_IPV4_STACK_IPV6_ADDRESS_EXCEPTION = "Option java.net.preferIPv4Stack is set to"
+                                                               + " true and an IPv6 address was provided in the config.";
+    private static final String NO_ADDRESSES_AVAILABLE_FOR_BINDING = "No addresses available for binding for URI: %s.";
```

Comment:
I personally don't think constants for strings that are used once are necessary and I would normally inline these.  That is completely personal preference and some people would disagree with me.  But if you choose to keep these as constants, anything with a %s or other formatter needs to have _FORMATTER or something else in the name saying it is a formatter and not a string